invenio_search/ext.py

```diff
@@ -79,6 +82,24 @@ def register_mappings(self, alias, package_name):
         :param alias: The alias.
         :param package_name: The package name.
         """
+        if ES_VERSION[0] == 2:
```

Comment:
Why is it done only for ES 2 and not ES5? ES5 will also be deprecated at some point.

Reply:
Let's add a comment:
```For backward compatibility reason, check the root directory for mappings.```